lib/hw_unique_key/Kconfig

```diff
@@ -4,6 +4,15 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+config HW_UNIQUE_KEY_LOAD
```

Comment:
I would like if we have a helptext here  if possible. Just to clarify that this is only needed for devices without a KMU.

Reply:
Added